### DISCUSSION TITLE
where is the RPC-demo which used rpc server  client?

### DISCUSSION THREAD

**Discussion by mingwei-learn** [OpenAMP/open-amp#385] [Q&A] [answered]
Hi:
    I want to use RPC  to develop my software between linux&Freertos on my zynq7020 board，I found something useful code on the github-openamplib。I hope to get a demo about rpc which used the new rpc_server_client code。 I tryed petalinux2020.1 petalinux2021.2，petalinux2022.1.  but the demos are same version  2018. the new demo 
      I  tryed build openamp-lib   by myself ，but something about openamp.shm post error。it seems that I need to set something about shm in device-tree.  I also can`t get help from UG1186 -v2021.
      Is there any new-RPC demo for me to easy used, or new  UG1186 support？

**Comment by arnopo** [comment, marked as answer]
Hello,
I suggest you to address the point to Xilinx support as it concerns the Xilinx porting of the open-AMP library.

**Reply by mingwei-learn** [reply]
> Hello, I suggest you to address the point to Xilinx support as it concerns the Xilinx porting of the open-AMP library.

is that openamp not supported by Xilinx? ok, I`m sorry.

**Reply by arnopo** [reply]
Companies usually adapt the open-amp and libmetal libraries and have their own build chain. Since you based your development on a Xilinx platform with a Xilinx user guide, this seems to be a better entry point for support. Secondly, if your problem is not related to a Xilinx specificity, feel free to come back to us.
